src/briefcase/platforms/web/static.py

```diff
@@ -0,0 +1,347 @@
+                    )
+                    css_file.write(f" * {package}::{'/'.join(path.parts[2:])}\n")
+                    css_file.write(
+                        " *******************************************************/\n\n"
+                    )
+                    css_file.write(wheel.read(filename).decode("utf-8"))
+
+    def build_app(self, app: AppConfig, **kwargs):
+        """Build the static web deployment for the application.
+
+        :param app: The application to build
+        """
+        self.logger.info("Building web project...", prefix=app.app_name)
+
+        if self.wheel_path(app).exists():
+            with self.input.wait_bar("Removing old wheels..."):
+                self.tools.shutil.rmtree(self.wheel_path(app))
+
+        self.wheel_path(app).mkdir(parents=True)
+
+        with self.input.wait_bar("Building app wheel..."):
+            try:
+                self.tools.subprocess.run(
+                    [
+                        sys.executable,
+                        "-u",
+                        "-m",
+                        "wheel",
+                        "pack",
+                        self.app_path(app),
+                        "--dest-dir",
+                        self.wheel_path(app),
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Unable to build wheel for app {app.app_name!r}"
+                ) from e
+
+        with self.input.wait_bar("Installing wheels for dependencies..."):
+            try:
+                self.tools.subprocess.run(
+                    [
+                        sys.executable,
+                        "-u",
+                        "-m",
+                        "pip",
+                        "wheel",
+                        "--wheel-dir",
+                        self.wheel_path(app),
+                        "-r",
+                        self.bundle_path(app) / "requirements.txt",
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Unable to install dependencies for app {app.app_name!r}"
+                ) from e
+
+        with self.input.wait_bar("Writing Pyscript configuration file..."):
+            with (self.project_path(app) / "pyscript.toml").open(
+                "w", encoding="utf-8"
+            ) as f:
+                f.write(f"name = '{app.formal_name}'\n")
+                f.write(f"description = '{app.description}'\n")
+                f.write(f"version = '{app.version}'\n")
+                f.write("\n")
+                f.write("autoclose_loader = true\n")
+                f.write("packages = [\n")
+                f.write(
+                    "\n".join(
+                        f'    "/{wheel.relative_to(self.project_path(app))}",'
```

Comment:
The toml library doesn't appear to always like these double quotes. On Linux, it doesn't seem to matter; on Windows, pyscript throws an error for `pyscript.toml`.

```
config supplied: name = 'mywebapp' description = 'My first application' version = '0.0.1' autoclose_loader = true packages = [ "/static\wheels\mywebapp-0.0.1-py3-none-any.whl", "/static\wheels\toga_core-0.3.0.dev36-py3-none-any.whl", "/static\wheels\toga_dummy-0.3.0.dev36-py3-none-any.whl", "/static\wheels\toga_web-0.3.0.dev36-py3-none-any.whl", "/static\wheels\travertino-0.1.3-py3-none-any.whl", ] is an invalid TOML and cannot be parsed: SyntaxError: w 7 | "/static\wheels\mywebapp-0.0.1-py3-none-any.whl", ^
```

Reply:
I've reversed the quoting so that we're using `"` consistently in output.

Reply:
Ok - I think we might need to move to a proper TOML serialiser rather than writing TOML as strings. Update incoming.

Reply:
I created #905 to reflect the need to ensure `pyproject.toml` is a proper TOML document.

Of note, while using `toml_w` here is nice....it only supports writing an entire TOML document; so, it wouldn't be useful for sending strings in to `cookiecutter` to create `pyproject.toml`.